### DISCUSSION TITLE
PaddleOCR/main/ppocr/model_train/kie

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14281] [Q&A]
# PaddleOCR/main/ppocr/model_train/kie

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/main/ppocr/model_train/kie.html

**Comment by MrJson1** [comment]
文档写得云里雾里的，到底是想让人看懂还是不想让人看懂，对小白一点都不友好
